charts/quickwit/templates/indexer-statefulset.yaml

```diff
@@ -50,7 +50,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-            {{- include "quickwit.environment" . | nindent 12 }}
+            {{- include "quickwit.metastore.environment" . | nindent 12 }}
```

Comment:
Thanks for adding this. Even though the description in https://github.com/quickwit-oss/helm-charts/issues/81 only mentions the searcher/indexer, I think the janitor and control plane should also get the metastore configuration.

Reply:
Okay, I will update this :)

Reply:
`quickwit.environment` is now unused with this change do you want to keep separated it for a future feature in an other PR ? Or i merge metastore definition in environment ?